install/helm/agones/templates/controller.yaml

```diff
@@ -13,6 +13,10 @@
 # limitations under the License.
 
 {{- $featureGates := include "agones.featureGates" . | fromYaml }}
+{{- $replicas := .Values.agones.controller.replicas }}
```

Comment:
Do we have to ensure that replicas is an int here? It would fail the `gt` checks if it isn't an int. 
But I guess that would mean relying on the helm check to give an typing error instead of us stating the error earlier. This also applies to validating if `replicas` is greater than 0, it will also give an typing error when comparing with a 0.

Reply:
I see I see. You are correct, there is a need to cast to int. This also means that we would need to check if values is 0 once it's casted. What is the behavior we want when the value is set to 0? Do we want it to default to 1?